src/frontend/Komponenter/Verktøy/SamværkalkulatorSide.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { SamværKalkulator } from '../../Felles/Kalkulator/SamværKalkulator';
+import { BodyShort } from '@navikt/ds-react';
 
 export const SamværkalkulatorSide: React.FC = () => {
-    return <SamværKalkulator />;
+    return <BodyShort>Under arbeid</BodyShort>;
```

Comment:
Er denne klar for å slettes?

Reply:
Done [66058ba](https://github.com/navikt/familie-ef-sak-frontend/pull/3003/commits/66058bad67ceba9ea4d5d74b91621fcd1407742c)